app/permissions/decidim/file_authorizations_handler/permissions.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Decidim
+  module FileAuthorizationHandler
+    # Defines the abilities related to surveys for a logged in admin user.
+    # Intended to be used with `cancancan`.
+    class Permissions < Decidim::DefaultPermissions
+      def permissions
+        return permission_action unless user
+        return Decidim::FileAuthorizationHandler::Admin::Permissions.new(user, permission_action, context).permissions if permission_action.scope == :admin
```

Comment:
Maybe I am missing something, but I don't see why we need to check for the Admin permissions in here. Could you explain the use case? Thanks!

Reply:
I've removed this file as there is no one is using it.